qiita_db/metadata_template/prep_template.py

```diff
@@ -666,3 +670,35 @@ def to_dataframe(self, add_ebi_accessions=False):
                 lambda sid: accessions[sid])
 
         return df
+
+    @property
+    def creation_timestamp(self):
```

Comment:
Isn't there a missing call to `creation_timestamp` where the object itself is created? And if so, should this property have a setter?

Reply:
Good question, as the default in the database is CURRENT_TIME, when the prep is inserted for the first time (creation) it will automatically add the CURRENT_TIMESTAMP so no need to actually update it (setter).

Reply:
Oh! Gotcha, thanks for clarifying. 😮